test/Database/Tables/ProductsWithSlashInColumnNames.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS [ProductsWithSlashInColumnNames];
+
+CREATE TABLE [ProductsWithSlashInColumnNames] (
+    [ProductId] [int] NOT NULL PRIMARY KEY,
+    [Name/Test] [nchar](100) NOT NULL,
+	[Cost/Test] [int] NOT NULL
```

Comment:
nit: Fix indentation
```suggestion
    [ProductId] [int] NOT NULL PRIMARY KEY,
    [Name/Test] [nchar](100) NOT NULL,
    [Cost/Test] [int] NOT NULL
```